pkg/streamingpromql/operators/aggregations/common.go

```diff
@@ -35,3 +37,27 @@ var AggregationGroupFactories = map[parser.ItemType]AggregationGroupFactory{
 //
 // Invalid combinations include exponential and custom buckets, and histograms with incompatible custom buckets.
 var invalidCombinationOfHistograms = &histogram.FloatHistogram{}
+
+func GetAggregationItemType(aggregation string) (parser.ItemType, bool) {
+	// The parser key is not exported, but the item types are. It's safe to assume the keys will continue to map their names onto their values.
```

Comment:
I'm not sure I follow this comment - are you trying to say that it's safe to assume that (for example) the string `sum` will continue to represent `parser.SUM`?

Reply:
Yes.

Reply:
How about rephrasing this to something like this then?

```suggestion
	// The aggregation names are not exported, but their item types are. It's safe to assume the names will not change.
```